test/LogicalPtrToInt/local_ptrtoint_64.cl

```diff
@@ -1,4 +1,4 @@
-// RUN: clspv %target %s -o %t.spv --hack-logical-ptrtoint
+// RUN: clspv %target %s -o %t.spv --hack-logical-ptrtoint -arch=spir64
```

Comment:
```suggestion
// RUN: clspv %s -o %t.spv --hack-logical-ptrtoint -arch=spir64
```

`%target` gets compiled to `-arch=spir` or `-arch=spir64` depending on the test build target.